### DISCUSSION TITLE
How can i create dynamic Table of Content ?

### DISCUSSION THREAD

**Discussion by miraculusik** [nvim-neorg/neorg#759] [Q&A] [answered]
```javascript
    @code norg
    = ToC Table of Contents
    @end
```
this code doesn't work for me. How can I create it ?

**Comment by max397574** [comment]
you did open that twice

**Reply by miraculusik** [reply]
how can i delete duplicate discussion ? i haven't seen any of delete option in discussion section.

**Comment by vhyrro** [comment, marked as answer]
The official syntax to create a TOC is the following:
```
.toc Table of Contents
```
But note that this syntax only constructs a table of contents during *export* (or at least that is its purpose). If you want an interactive TOC then you can use `:Neorg toc` in any `norg` buffer of your choice :)

Where did you get the old syntax from? Maybe there's some old bit of documentation that I overlooked and that I should update :p

**Reply by miraculusik** [reply]
thank you. `:h Neorg` I read this help page, at the bottom of the this page it was showing me creating it like this
